thispastwinter/To-Do
# JKA-37: ButtonCheckCircle test complete

## Description

Test for the ButtonCheckCircle Component

### 👩‍🔬 Test Instructions

1. Check codebase.
2. Run test for CheckButtonCircle and make sure it passes.
3. Ensure that test now contains jest testing and not just snapshots.

## 🔎 Reviewer Checklist

> Checked off by the PR **Reviewers**

### Required

> These always need to be checked


>### Roles & Responsibilities
>
>#### 👨‍💻 Justin Klaas
>
>- Initiator of this PR (be sure to set in GitHub UI)
>- Addresses feedback and change requests
>- Merges PR once approved (usually deletes branch unless `develop` or `release`)
>
>#### 👩‍💻 Blake Tarter/Rob LaFeve
>
>- Invited to review PR by **Assignee** (via GitHub UI)
>- Is expected to complete a review and address followup
